dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -305,6 +306,11 @@ def run_validation(self, job):
             error_list.write_all_row_errors(job_id)
             # Update error info for submission
             populate_submission_error_info(submission_id)
+
+            # Update last validated date
+            job.submission.last_validated = datetime.utcnow()
+            sess.commit()
```

Comment:
I can't comment higher up because GitHub, but do we need the sess.commit on line 304 if we're committing right here?